templates/ctl-scaffold/test/Main.purs

```diff
@@ -6,22 +6,33 @@ import Contract.Prelude
 
 import Contract.Config as Contract.Config
 import Contract.Monad as Contract.Monad
+import Contract.Test.Mote as Contract.Test.Mote
+import Contract.Test.Plutip (PlutipTest, testPlutipContracts)
 import Contract.Test.Plutip as Contract.Test.Plutip
-import Contract.Wallet as Contract.Wallet
 import Data.BigInt as BigInt
 import Data.UInt as UInt
+import Effect.Aff (Milliseconds(Milliseconds))
+import Mote (test)
 import Scaffold as Scaffold
+import Test.Spec.Runner (defaultConfig)
 
+-- Run with `spago test --main Test.Scaffold.Main`
```

Comment:
`npm run test`

Reply:
I also changed this in `Test.Ctl.Plutip`, from where I copied the command.